alicloud/service_alicloud_ecs.go

```diff
@@ -766,3 +766,31 @@ func (s *EcsService) QueryInstanceAllDisks(id string) ([]string, error) {
 	}
 	return ids, nil
 }
+
+func (s *EcsService) DescribeLaunchTemplate(id string) (set ecs.LaunchTemplateVersionSet, err error) {
+	req := ecs.CreateDescribeLaunchTemplateVersionsRequest()
+	req.LaunchTemplateId = id
+
+	raw, err := s.client.WithEcsClient(func(ecsClient *ecs.Client) (interface{}, error) {
+		return ecsClient.DescribeLaunchTemplateVersions(req)
+	})
+	if err != nil {
+		if IsExceptedError(err, "InvalidLaunchTemplate.NotFound") {
+			err = GetNotFoundErrorFromString(fmt.Sprintf("The specified Launch Template is not found by id %s", id))
+			return
+		}
+		err = WrapErrorf(err, DefaultErrorMsg, id, req.GetActionName(), AlibabaCloudSdkGoERROR)
+		return
+	}
+	resp := raw.(*ecs.DescribeLaunchTemplateVersionsResponse)
+	if resp == nil {
+		err = fmt.Errorf("describe launch template version got nil")
+		return
+	}
+	if len(resp.LaunchTemplateVersionSets.LaunchTemplateVersionSet) == 0 {
+		err = GetNotFoundErrorFromString(fmt.Sprintf("The specified Launch Template is not found by id %s", id))
+		return
```

Comment:
The line 791 should be `err = WrapErrorf(Error(GetNotFoundMessage("LaunchTemplate", id)), NotFoundMsg, ProviderERROR)`